commons/src/lib.rs

```diff
@@ -172,19 +196,44 @@ mod tests {
     #[test]
     fn test_validate_content_type() {
         let mut headers = actix_web::http::HeaderMap::new();
-        validate_content_type(&headers, "application/json").unwrap(); // if the request leaves Accept empty, we can return whatever we want
+
+        let content_type: Vec<actix_web::http::HeaderValue> =
+            vec![header::HeaderValue::from_str("application/json").unwrap()];
+        let version = validate_content_type(&headers, content_type.clone()).unwrap(); // if the request leaves Accept empty, we return the minimum supported cincinnati version
+        assert_eq!(version, MIN_CINCINNATI_VERSION.to_string());
+
         headers.insert(
             header::ACCEPT,
             //"application/json, text/*; q=0.2".parse().unwrap(), // prefer JSON, but also accept any text/* after an 80% markdown in quality.  FIXME: needs a smarter parser in validate_content_type
             "application/json".parse().unwrap(),
         );
-        validate_content_type(&headers, "application/json").unwrap();
+        let version = validate_content_type(&headers, content_type.clone()).unwrap();
+        assert_eq!(version, MIN_CINCINNATI_VERSION.to_string());
+
         headers.insert(
             // FIXME: drop once validate_content_type gets a smarter parser and the previous insert can include the text/* entry
             header::ACCEPT,
             "text/*".parse().unwrap(),
         );
-        validate_content_type(&headers, "text/plain").unwrap();
-        validate_content_type(&headers, "image/png").unwrap_err();
+        let text_type: Vec<actix_web::http::HeaderValue> =
+            vec![header::HeaderValue::from_str("text/plain").unwrap()];
+        let version = validate_content_type(&headers, text_type).unwrap();
+        assert_eq!(version, MIN_CINCINNATI_VERSION.to_string());
+
+        let image_type: Vec<actix_web::http::HeaderValue> =
+            vec![header::HeaderValue::from_str("image/png").unwrap()];
+        validate_content_type(&headers, image_type).unwrap_err();
+
+        headers.insert(
+            // FIXME: drop once validate_content_type gets a smarter parser and the previous insert can include the text/* entry
+            header::ACCEPT,
+            "application/vnd.redhat.cincinnati.v1+json".parse().unwrap(),
+        );
+        let cincinnati_v1: Vec<actix_web::http::HeaderValue> =
+            vec![
+                header::HeaderValue::from_str("application/vnd.redhat.cincinnati.v1+json").unwrap(),
+            ];
+        let version = validate_content_type(&headers, cincinnati_v1).unwrap(); // if the request leaves Accept empty, we can return whatever we want
+        assert_eq!(version, 1);
```

Comment:
```suggestion
        assert_eq!(version, "1");
```
Comparing strings to strings